src/engine/Input/PointerComponent.ts

```diff
@@ -0,0 +1,22 @@
+import { Component } from '../EntityComponentSystem/Component';
+
+/**
+ * Add this component to optionally configure how the pointer
+ * system detects pointer events.
+ *
+ * By default the collider shape is used and graphics bounds is not.
+ *
+ * If both collider shape and graphics bounds are enabled it will fire events if either or
+ * are intersecting the pointer.
+ */
+export class PointerComponent extends Component<'ex.pointer'> {
+  public readonly type = 'ex.pointer';
+  /**
+   * Use any existsing Collider component geometry for pointers
```

Comment:
Maybe should expand on why use this vs. other option

Also, can _both_ be `true`?

Reply:
They can both be true, your collider and graphics bounds could cover different areas.

Good point, I'll add some additional detail about the why.